src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: small;
+  font-style: bold;
 }
+
+
+li {
+  padding: 23px 0;
+  margin-right: 21px;
+}
```

Comment:
1. apply styles to class selectors instead of tag ones
2. `li` should have only margin to get an indent between them (but last child shouldn't have margin). You can use `:not`